Arcane Engine Core/src/input/JoystickInputData.h

```diff
@@ -0,0 +1,78 @@
+/*
+	Author: Mohamed Kazma
```

Comment:
Not sure if we are doing this for every file?

Reply:
old habits die hard it seems